cmd/ipi-deprovision/ipi-deprovision.sh

```diff
@@ -53,6 +53,24 @@ EOF
   echo "will deprovision GCE cluster ${infraID} in region ${region}"
 done
 
+libvirt_p_cluster_age_cutoff="$(TZ=":America/New_York" date --date="-11 hours" '+%Y-%m-%dT%H:%M%z')"
+echo "deprovisioning clusters with creation time before ${libvirt_p_cluster_age_cutoff} in libvirt-p"
+for network in $(ssh ${bastion_p} virsh net-list --all --name | grep -v default) ; do
```

Comment:
What sets `"${bastion_p}"`?

Reply:
nit: quote your variable expansions

Reply:
i was hoping that'd be something we could add to the cleanup run. is that possible? i'm not familiar with how these values get set. i noticed that `CLUSTER_TTL` was set somewhere.

Reply:
Sure, we could -- is it a static IP?

Reply:
yah it's static. for the ci it's hard-coded into the release job:
https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/release/openshift-release-release-4.3-periodics.yaml#L6912